.github/workflows/codeql.yml

```
@@ -54,6 +54,8 @@ jobs:
         with:
           languages: ${{ matrix.language }}
           build-mode: ${{ matrix.build-mode }}
+          paths-ignore: |
```

actionlint (1.7.4)

input "paths-ignore" is not defined in action "github/codeql-action/init@v3". available inputs are "build-mode", "config", "config-file", "db-location", "debug", "debug-artifact-name", "debug-database-name", "external-repository-token", "languages", "matrix", "packs", "queries", "ram", "registries", "setup-python-dependencies", "source-root", "threads", "token", "tools", "trap-caching"

(action)